lib/Drush/UpdateService/StatusInfoDrush.php

```diff
@@ -370,7 +379,7 @@ private function calculateProjectUpdateStatus($project_release_info, &$project_d
     // Bail out early.
     if (!isset($project_data['recommended'])) {
       $project_data['status'] = DRUSH_UPDATESTATUS_UNKNOWN;
-      $project_data['reason'] = t('No available releases found');
+      $project_data['reason'] = dt('No available releases found');
```

Comment:
This should be fixed regardless. I was about to submit a separate pull request and can do so if this is not really something considered for addition, although I hope it is.